### PR TITLE
Update sql for config flow

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Update markdown for config flow for sql integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68700
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sql.markdown [hunk 1]
```diff
@@ -16,56 +18,13 @@ ha_platforms:
...
-    unit_of_measurement:
-      description: Defines the units of measurement of the sensor, if any.
-      required: false
-      type: string
-    value_template:
-      description: Defines a template to extract a value from the payload.
-      required: false
-      type: template
-{% endconfiguration %}
+The SQL integration will by connect to default recorder if Database URL is not specified.
```
Comment (dgomes) [inline source/_integrations/sql.markdown]:
```suggestion
The SQL integration will connect to default recorder if Database URL is not specified.
```

File: source/_integrations/sql.markdown [hunk 2]
```diff
@@ -79,117 +38,56 @@ In this section, you find some real-life examples of how to use this sensor.
 
 This example shows the previously *recorded* state of the sensor `sensor.temperature_in`.
 
-```yaml
```
Comment (dgomes) [inline source/_integrations/sql.markdown]: why are you erasing the example sensor (which is not SQL sensor) ?

File: source/_integrations/sql.markdown [hunk 3]
```diff
@@ -79,117 +38,56 @@ In this section, you find some real-life examples of how to use this sensor.
...
-sensor:
-  - platform: sql
-    queries:
-      - name: Temperature in
-        query: "SELECT * FROM states WHERE entity_id = 'sensor.temperature_in' ORDER BY state_id DESC LIMIT 1;"
-        column: "state"
-```
-
-Note that the SQL sensor state corresponds to the last row of the SQL result set.
+Use `state` as column for value.
```
Comment (dgomes) [inline source/_integrations/sql.markdown:54]: I would provide a screenshot *or* description of what goes in where in the config flow

Comment (frenck) [reply source/_integrations/sql.markdown:54]: >  I would provide a screenshot or description of what goes in where in the config flow

We generally don't provide screenshots of configuration flows.

**Review by dgomes — APPROVED** [review]

**Review by bdraco — APPROVED** [review]

**Pull request merged**